test/validators/required-if-not.spec.js

```diff
@@ -0,0 +1,255 @@
+import noes from 'noes';
+import { expect } from 'chai';
+import validator from '../../lib';
+import requiredIf from '../../lib/validators/required-if';
+
+describe('Required if not validator', () => {
+  context('with validator wrapper', () => {
+    context('when given normal parameters', () => {
+      const simpleRules = {
+        address: ['requiredIfNot:hasHome:Yes']
+      };
+
+      it('should success if hasHome is No and address is defined', () => {
+        const result = validator.validate(simpleRules, {
+          hasHome: 'No',
+          address: 'somewhere over the rainbow'
+        });
+        const err = result.messages;
+
+        expect(result.success).to.equal(true);
+        expect(err).to.not.have.property('address');
+      });
```

Comment:
this test case is good, it is solid and meet the requirement, but your example somehow somewhat hurts my brain, address is required if has home is not yes (which is no). maybe you should switch the case.

Anyway, good work!

Reply:
WKWK
done ya Ko (y)